build/build.sh

```diff
@@ -41,7 +41,8 @@ fi
 
 export CGO_ENABLED=0
 
-go build -a -installsuffix cgo 	\
+go build \
+    ${GOBUILD_FLAGS} \
```

Comment:
you need to set a default in case this env value is not set (that's why this is failing CI)